admob/NativeAdvancedExample/app/src/main/java/com/google/example/gms/nativeadvancedexample/MainActivity.java

```diff
@@ -147,7 +147,15 @@ public void onVideoEnd() {
             adView.getPriceView().setVisibility(View.VISIBLE);
             ((TextView) adView.getPriceView()).setText(nativeAppInstallAd.getPrice());
         }
-
+        
+        NativeAd.Image logoImage = nativeAppInstallAd.getIcon();
```

Comment:
Since the app icon is a [required asset](https://support.google.com/admob/answer/6240809), please move this code up to where the other required assets are set.